src/jti_background.js

```diff
@@ -130,20 +162,24 @@ function fetchDefaultTemplates (callback) {
 }
 
 function setDefaultTemplates (callback) {
-    fetchDefaultTemplates(function (status, message, data) {
-        if (status) {
-            data.templates = JSONtoTemplateData(data.templates);
-            saveTemplates(data, callback);
-        } else {
-            callback(false, message);
-        }
+    chrome.storage.sync.get(StorageID, function (data) {
```

Comment:
Why are you calling `chrome.storage.sync.get(StorageID, function (data) {` ?
`fetchDefaultTemplates` already returns `status, message, data`
So one of the 'data' objects your accessing is being overwritten.

Reply:
:+1: